map2annotation/fetch_map2anno_tools.sh

```diff
@@ -1,17 +1,10 @@
 #!/bin/bash
 MAP2ANNO_DIR=$(cd $(dirname "$0") && pwd)
 
-##### build hacked checkstyle
```

Comment:
Can you explain how the hacked checkstyle is related to the multiDeclRefactor?  Why are these both in the same code review?